### PR TITLE
Build k8s takeunder

### LINKED ISSUE #3049: update homepage takeunder - Dev
please update the takeunder section.
keep Openstack and replace Product of the month with Kubernetes.
Copy and link addresses in the brief: https://docs.google.com/document/d/1Kr_Y_1LdB_AjqeVEdlAiEWVuNjj6o12WCQte_Uj1cSk/edit

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update homepage takeunder

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Check that first homepage takeunder is Kubernetes and matches [design](https://user-images.githubusercontent.com/36884067/39246185-b625fafa-488d-11e8-8001-8d30905577a3.jpg) and [copy doc](https://docs.google.com/document/d/1Kr_Y_1LdB_AjqeVEdlAiEWVuNjj6o12WCQte_Uj1cSk/edit#)

## Issue / Card

Fixes #3049

### CODE REVIEW

File: templates/takeovers/takeunders/_ubuntu-product-month.html [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-        <img src="https://assets.ubuntu.com/v1/b0bd120e-UPM-Illustration.svg" width="180" alt="" />
+        <img src="{{ ASSET_SERVER_URL }}1db850da-K8s+logo.svg" width="180" alt="" />
       </div>
-      <div class="col-8">
-        <h3><a href="https://pages.ubuntu.com/ubuntu_product_month.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Ubuntu Product Month takeunder', 'eventLabel' : 'Find out more', 'eventValue' : undefined });">Ubuntu product month omnibus&nbsp;&rsaquo;</a></h3>
-        <p>
-          Learn about OpenStack, LXD, Juju, and Snaps from&nbsp;the developers and teams behind them
-        </p>
+      <div class="col-8 u-no-margin--top">
+        <h3><a href="https://pages.ubuntu.com/kubernetes-month.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Ubuntu Product Month takeunder', 'eventLabel' : 'Find out more', 'eventValue' : undefined });">Watch our Kubernetes webinars&nbsp;&rsaquo;</a></h3>
```
Comment (anthonydillon) [inline templates/takeovers/takeunders/_ubuntu-product-month.html]: Please update `eventLabel` to match the button text

**Comment by anthonydillon** [comment]
There is no way for UX and design to review this branch. I would suggest they could review once switching to the takeover.

**Comment by anasereijo** [comment]
@steverydz 
please amend the copy: we have a > instead of ' - canonical's and can you please add a full stop?
thanks
screen shot 2018-04-25 at 14 28 10

**Comment by frankiegranato** [comment]
Sorry -  
Little text correction to sub-title (add the word 'about'). The copy should read: 

"A series of on-demand webinars to learn more about Canonical’s Distribution of Kubernetes."

**Comment by Caleb-Ellis** [comment]
@steverydz Is this supposed to me merging with master?

**Comment by steverydz** [comment]
@Caleb-Ellis Yes, but we're going to merge `release/18.04` into master first

**Comment by steverydz** [comment]
@Caleb-Ellis Fixed. Actually this can be merged into `release/18.04` as that hasn't yet been merged.

**Pull request merged**